2.1_first_module.ipynb

```diff
@@ -241,15 +381,64 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
+   "execution_count": 9,
```

Comment:
Looks like the second commit is the exercise-in-progress...

Reply:
Sorry about that, shouldn't include this file in the commit, my mistake.